aiida_mlip/calculations/train.py

```diff
@@ -89,6 +98,18 @@ def define(cls, spec: CalcJobProcessSpec) -> None:
             required=True,
             help="Config file with parameters for training",
         )
+
+        spec.input(
+            "fine_tune",
+            valid_type=Bool,
+            required=False,
+            default=lambda: Bool(False),
+            help="Fine-tuning of a model",
+        )
+        spec.input(
+            "model", valid_type=ModelData, required=False, help="Model to fine-tune"
```

Comment:
I think this should be the [model type](https://github.com/ACEsuit/mace/blob/main/mace/tools/arg_parser.py#L92), which I don't think it unique to fine tuning

Reply:
is it also a keyword in the normal training?

Reply:
Yes, I think so.

This might be fine as is, depending on how it's used.

`model` in the yaml config that's passed to MACE via janus is the model type, but is this just used as the actual model itself, or is it supposed to be the value in the yaml?

(If we do eventually add options other than via the config, having different meanings could make things more complicated)

Reply:
changed the keyword tofoundation_model